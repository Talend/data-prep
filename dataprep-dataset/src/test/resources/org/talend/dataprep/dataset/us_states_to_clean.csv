Postal;State;Capital;MostPopulousCity

al;Alabama;Montgomery;Birmingham city
--;--;--;--
AK;Alaska;Juneau;Anchorage
AZ;ARIZONA;PHOENIX;PHOENIX
AR;Arkansas;Little Rock;Little Rock
CA;California
CO;Colorado;Denver;Denver
