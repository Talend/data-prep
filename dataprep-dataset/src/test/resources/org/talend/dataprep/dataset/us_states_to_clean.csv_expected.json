{
  "records": [
    {
      "0000": "al",
      "0001": "Alabama",
      "0002": "Montgomery",
      "0003": "Birmingham city"
    },
    {
      "0000": "--",
      "0001": "--",
      "0002": "--",
      "0003": "--"
    },
    {
      "0000": "AK",
      "0001": "Alaska",
      "0002": "Juneau",
      "0003": "Anchorage"
    },
    {
      "0000": "AZ",
      "0001": "ARIZONA",
      "0002": "PHOENIX",
      "0003": "PHOENIX"
    },
    {
      "0000": "AR",
      "0001": "Arkansas",
      "0002": "Little Rock",
      "0003": "Little Rock"
    },
    {
      "0000": "CA",
      "0001": "California",
      "0002": "",
      "0003": ""
    },
    {
      "0000": "CO",
      "0001": "Colorado",
      "0002": "Denver",
      "0003": "Denver"
    }
  ]
}
